File: src/main/java/com/example/resthospital/Doctor/Doctor.java
package com.example.resthospital.Doctor;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Doctor
{
  @JsonProperty("cpr_doctor")
  int cpr_doctor;

  @JsonProperty("doctor_name")
  String doctor_name;

  @JsonProperty("doctor_age")
  int doctor_age;

  @JsonProperty("speciality")
  String speciality;

  @JsonProperty("city")
  String city;

  @JsonProperty("gender")
  String gender;

  @JsonProperty("house_number")
  int house_number;

  @JsonProperty("street")
  String street;

  @JsonProperty("postal_code")
  int postal_code;

  @JsonProperty("phone")
  int phone;

}

File: Models/Doctor.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace hospitalGQL.Models
{
    public class Doctor
    {
        [Key] 
        public long cpr_doctor { get; set; }

       
        public string doctor_name { get; set; }
         
        public long doctor_age{ get; set; }
      
        public string speciality { get; set; }
        
        public string city  { get; set; }
        
        public string gender  { get; set; }
        
        public long house_number { get; set; }
        
        public string street  { get; set; }
      
        public long postal_code  { get; set; }
       
        public long phone  { get; set; }
        public ICollection<Patient> Patients { get; set; } = new List<Patient>();

    }
}